packages/auth-frontend-utils/package.json

```diff
@@ -25,8 +26,7 @@
     "@babel/core": "7.19.1",
     "@babel/preset-react": "7.18.6",
     "@types/auth0-js": "9.14.6",
-    "@types/jest": "26.0.24",
-    "@types/node": "14.18.26",
```

Comment:
Remove to see if we get TS issues in FE so we can see where libraries need to be added